zookeeper/init.sls

```diff
@@ -33,15 +33,8 @@ install-zookeeper:
     - group: root
     
 zookeeper-home-link:
-  alternatives.install:
-    - link: {{ zk.alt_home }}
-    - path: {{ zk.real_home }}
-    - onlyif: test -d {{ zk.real_home }} && test ! -L {{ zk.alt_home }}
```

Comment:
This line is little bit wrong, that's why it doesn't work reliably. The command should be:
```bash
test -d {{ zk.real_home }} && test ! -L /etc/alternatives/zookeeper-home-link
```
It would install the alternative only if it doesn't exist yet.